src/ralph/data_center/admin.py

```diff
@@ -37,6 +37,7 @@
 from ralph.data_center.models.virtual import Database, VIP
 from ralph.data_center.views.ui import DataCenterAssetSecurityInfo
 from ralph.data_importer import resources
+from ralph.dns.views import DnsView
```

Comment:
DNS is abbreviation, so IMO (:sunglasses:) we should use capital letters for it (ex. `DNSView`)